plugins/hydra_ray_launcher/example/conf/ray_mode/local.yaml

```diff
@@ -0,0 +1 @@
+random_seed: 1
```

Comment:
why is random_seed in ray_mode=local?
what does it have to do with the ray_mode?